style/hypertext.rst

```diff
@@ -0,0 +1,70 @@
+=========
+Hypertext
+=========
+
```

Comment:
Shouldn't there be two lines after each section?

Reply:
good catch! thx